vowpalwabbit/parse_args.cc

```diff
@@ -1598,7 +1598,29 @@ void cmd_string_replace_value(std::stringstream*& ss, std::string flag_to_replac
   }
 }
 
-char** get_argv_from_string(std::string s, int& argc)
+char** to_argv_escaped(std::string const& s, int& argc)
+{
+  char* c = calloc_or_throw<char>(s.length() + 3);
```

Comment:
Since you will be reprocessing the escaped tokens anyways, do you need to start with copying the string with the prefix token? Could you just inject it right before the for loop below?

In other words:

```
char** argv = calloc_or_throw<char*>(tokens.size() + 1);
argv[0] = calloc_or_throw<char>(2); //or even point it to a constant string,
                                    //though that might run into cleanup issues
```